src/util/ChartUtils.ts

```diff
@@ -1330,6 +1338,35 @@ export const getTooltipItem = (
   };
 };
 
+export function getTooltipEntry({
```

Comment:
This no longer depends on `displayName` which will enable us to switch to functional components.